common/on_actions/imperia_on_capitulation.txt

```diff
@@ -0,0 +1,58 @@
+﻿# Root = Country
+on_imperia_capitulation = {
+	effect = {
+		## Gutter German Revolution
+		# If the Gutter King lost
+		if = {
+			limit = {
+				has_variable = accepted_gutter_crown
+				NOT = {
+					has_journal_entry = je_imperia_frankfurt_german_consolidation_gutter_king
+				}
+				has_global_variable = frankfurt_parliament_civil_war_happened
```

Comment:
One of these variables need to be reset in this effect, otherwise this might trigger again when the gutter king loses again to something completely unrelated.

Reply:
Both of them gets cleaned up in `imperia_frankfurt_parliament_clean_variables`, which fires from on_monthly_pulse_country on_action line 134. Conditions are basically that no more Gutter Germany shenanigans is happening.